client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -1526,6 +1526,12 @@ public void onFailure(Throwable e) {
         new RpcResponseCallback() {
           @Override
           public void onSuccess(ByteBuffer response) {
+            // Compatible with lower versions of Server
```

Comment:
could you please add more detail info here? e.g., JIRA tickets, versions, and why it can be treated as success